src/Servers/IIS/IIS/test/IIS.FunctionalTests/Http2TrailersResetTests.cs

```diff
@@ -240,6 +240,40 @@ public async Task Reset_BeforeResponse_Resets()
                 .Build().RunAsync();
         }
 
+        [ConditionalFact]
+        [RequiresNewHandler]
+        [MinimumOSVersion(OperatingSystems.Windows, WindowsVersionForTrailers)]
+        public async Task Reset_ConnectionRequestClose()
```

Comment:
@Tratcher I couldn't run IIS tests on my machine. I'm not sure if this test is correct.

Reply:
I'll verify it locally, it still requires a preview build of windows.

Reply:
```suggestion
        public async Task RequestClose_SendsGoAway()
```